_test/test/serve_integration_test.dart

```diff
@@ -157,4 +169,48 @@ void main() {
             .close();
     expect(await utf8.decodeStream(ddcFileResponse), contains('"goodbye"'));
   });
+
+  test('should serve files in parallel', () async {
+    await startServer(buildArgs: [
+      'web',
+      '--build-filter',
+      'web/sub/main.dart.js',
+      '--verbose',
+      '--define',
+      'build_web_compilers|ddc=generate-full-dill=true',
+    ]);
+
+    addTearDown(() async {
+      await stopServer();
+    });
+
+    Future<void> read(String path) async {
+      try {
+        var request = httpClient.get('localhost', 8080, path);
+        var response = await (await request).close();
+        expect(
+          response.statusCode,
+          HttpStatus.ok,
+          reason: '$path ${response.reasonPhrase}',
+        );
+        await response.drain();
+      } catch (e, s) {
+        print('Test error on: $e:$s');
```

Comment:
Why do we need this?

Reply:
Oh forgot to remove, thanks!

Reply:
replaced by 'fail' with extra information